.eslintrc.yml

```diff
@@ -0,0 +1,84 @@
+parser: "@typescript-eslint/parser"
```

Comment:
*Comment unrelated to line number*

This is the `eslintrc` we are using on the Wallet Service